sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -347,6 +347,7 @@ IShellFolder_GetDisplayNameOf(
 
 /* Flags for IShellFolder_GetDisplayNameOf */
 #define SFGDNO_RETRYWITHFORPARSING 1
+#define SFGDNO_RETRYALWAYS 0x80000000
```

Comment:
indent the values (as done for the other defines above in the file)

Reply:
Fixed.